Description: fix Doxygen documentation settings for Debian
Author: Ghislain Antony Vaillant <ghisvail@gmail.com>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/clBLAS.doxy
+++ b/doc/clBLAS.doxy
@@ -52,7 +52,7 @@
 # If a relative path is entered, it will be relative to the location 
 # where doxygen was started. If left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = ..\..\bin\clBLAS.doxy
+OUTPUT_DIRECTORY       = 
 
 # If the CREATE_SUBDIRS tag is set to YES, then doxygen will create 
 # 4096 sub-directories (in 2 levels) under the output directory of each output 
@@ -1223,7 +1223,7 @@
 # installing MathJax.  However, it is strongly recommended to install a local 
 # copy of MathJax from http://www.mathjax.org before deployment.
 
-MATHJAX_RELPATH        = http://www.mathjax.org/mathjax
+MATHJAX_RELPATH        = /usr/share/javascript/mathjax
 
 # The MATHJAX_EXTENSIONS tag can be used to specify one or MathJax extension 
 # names that should be enabled during MathJax rendering.
